frontend/lindsay-wildlife-treasure-hunt-frontend/src/App.css

```diff
@@ -1,7 +1,15 @@
+@font-face {
+  font-family: 'Avenir Book';
+  src: url('./src/fonts/AvenirLTStd-Book.otf')
```

Comment:
just want to make sure... did you test these URLs via `docker-compose`. You can test by running `run.sh`

Reply:
No, I just tested with the React dev server... which run.sh are you referring to?

Reply:
update: tested docker-compose locally and font loading works :)